subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/ConsoleCompositeBuildFunctionalTest.groovy

```diff
@@ -17,18 +17,18 @@
 package org.gradle.internal.logging
 
 import org.gradle.integtests.fixtures.AbstractConsoleFunctionalSpec
+import org.gradle.internal.SystemProperties
 import spock.lang.Ignore
 
-@Ignore("Build fails and passes depending on timing")
+//@Ignore("Build fails and passes depending on timing")
```

Comment:
I am not sure if the other test in there will actually pass. For me it was somewhat flaky.